tools/sylph/tool_data_table_conf.xml.test

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<tables>
+    <!-- Locations of Sylph database in the required format -->
+    <table name="sylph_databases" comment_char="#">
```

Comment:
How will you get the cached database? With a data manager?

Reply:
I explained it in the readme we put in it. The db and metadata files have to be associated with one another correctly, so the endpoint of the data table is to a directory where both the db and metadata files will be stored. There is a page on the secondary sylph repo with all of the dbs and metadata files. We're also wrapping a tool to create those files next